src/styled/index.css

```diff
@@ -1,11 +1,2 @@
+/* Fonts */
 @import url('https://fonts.googleapis.com/css2?family=Teko:wght@300;500;600&display=swap');
```

Comment:
is it possible to make this import a globalStyle then?

Reply:
styled components prefer import fonts by link tag in head tag of index.html or we can add to project next library such as react-helmet to inject fonts